1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,46 @@
 function readingList(books) {
   // Write your code here...
+  let divEl = document.getElementById("content");
+  let ulEl = document.createElement("ul");
+  divEl.appendChild(ulEl);
+
+  for (let book of books) {
+    let liEl = document.createElement("li");
+    let para = document.createElement("p");
+    para.innerText = `${book.title} by ${book.author}`;
+    liEl.appendChild(para);
+    let image = document.createElement("img");
+    image.src = book.bookCoverImage;
+    liEl.appendChild(image);
+    if (book.alreadyRead) {
+      liEl.style.backgroundColor = "green";
+    } else {
+      liEl.style.backgroundColor = "red";
+    }
+    ulEl.appendChild(liEl);
+  }
```

Comment:
this is a clear solution and works really well Doris :) you could go further and extract the code in your for loop into a function and call it something like `createBookElement` then you have a reusable function that can be used elsewhere (for example imagine if you wanted to add a new book to the list, a function like this would be quite useful)